app.js

```diff
@@ -1,2 +1,148 @@
-/* add your code here */
+document.addEventListener('DOMContentLoaded', () => {
+    // function will go here
+    const stocksData = JSON.parse(stockContent);
+    const userData = JSON.parse(userContent);
+    const deleteButton = document.querySelector('#btnDelete');
+    const saveButton = document.querySelector('#btnSave');
+
+    generateUserList(userData);
+
+    deleteButton.addEventListener('click', (event) => {
+        
+        event.preventDefault();
+    
+        
+        const userId = document.querySelector('#userID').value;
+        const userIndex = userData.findIndex(user => user.id == userId);
+        
+        userData.splice(userIndex, 1);
+        
+        generateUserList(userData, stocksData);
+      })
+
+      saveButton.addEventListener('click', (event) => {
+      e.preventDefault();
```

Comment:
This function isn't working correctly. Here, we have the callback argument defined as `event` but we are referencing it in the body of the function as `e`.

```suggestion
      event.preventDefault();
```